tests/unit/Core/Framework/Rule/CustomFieldRuleTest.php

```diff
@@ -66,325 +67,514 @@ public function testMatch(
     public static function customFieldRuleData(): array
     {
         return [
-            'custom field null / rendered field value true/ bool type/ EQ operator / not matching' => [
+            'test equals with bool type and missing value (field value true)' => [
```

Comment:
You don't need to add test everywhere, since PHPUnit collects them as "subtests". I'd rather use a fitting testname, which is better than "testMatch" 😉

Reply:
btw: How is the duration of thes test? Is really every case necessary? Don't forget those have an impact to the pipeline duration as well. So this should be considered

Reply:
it is a unit test, it should be very fast

Reply:
ah, of course. Not an integration test :D